R/dependencies.R

```diff
@@ -1655,53 +1655,27 @@ renv_dependencies_report <- function(errors) {
     stopf(fmt)
   }
 
-  renv_condition_signal("renv.dependencies.error", problems)
+  renv_condition_signal("renv.dependencies.problem", problems)
```

Comment:
I think "problem" is better than "error" because it doesn't imply that it's an error condition.